bin/ubuntu-core-initramfs

```diff
@@ -616,6 +611,58 @@ def create_initrd_pkg_list(dest_dir, sysroot):
                             pkgs).decode("utf-8")
         pkg_list.write(out)
 
+def verify_missing_dlopen(destdir, libdir):
```

Comment:
This would need a comment saying what it does and why. Maybe also add an example json and maybe a link to where the format is defined. Naming the libraries that have this section would be good too (I see libsystemd-shared-256.so, but not sure if others have it).